### ISSUE TITLE
Manage concurrency

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter-exporter-sinatra#19]
Looks like if we constantly refresh `status.json` we may increase the concurrency count. 

```
I'd set concurrency to 2 (instead of 80 lol) and republishing on GCloud

https://export.mapknitter.org/export?url=https://mapknitter.org/maps/ceres--2/warpables.json&scale=30 now outputs /1556636097/status.json, which is a real json file!

ohhhhhh oops, concurrency has to be set differently because now after refreshing status.json 2x, it starts a new container

increased to 80 concurrency again
```

Perhaps if the requests come in slow, it's not as bad an issue... is concurrency not the total # of requests but the total at the same time? probably. 

Alternatively, is there any way to only spawn new instances if we are running concurrent export processes, instead of serving multiple HTTP requests? Like, only count concurrency on the export routes?

Another issue (which will be solved if we require cloud storage after #17) is that different instances will store different `status.json` files, in different file systems.

**Comment by icarito** [comment]
As it stands today we are running on GKE which handles this by CPU load (it's tweakable).
I've set it up with min 1 max 4 containers - can't have 0.

> Another issue (which will be solved if we require cloud storage after #17) is that different instances will store different status.json files, in different file systems.

Indeed, fixed in #17.
